### ISSUE TITLE
Build new classes about intermediate HTML/CSS

### ISSUE THREAD

**Issue by felipez3r0** [felipez3r0/openclasses#9]
Build around 20 classes about intermediate HTML/CSS subject

**Comment by abhishek71994** [comment]
can it involve a whole project using which intermediate HTML/CSS can be taught?

**Comment by felipez3r0** [comment]
Yes, if you follow the method no problem 😄

**Comment by abhishek71994** [comment]
The length has to be 4 hours?

**Comment by felipez3r0** [comment]
Yes, we need 18 - 20 classes around learning intermediate HTML/CSS and around 4 hours each class following th structure in the [examples](https://github.com/felipez3r0/openclasses/tree/master/Examples/Webdesign/Class_1). If you want to help, just start a new folder inside examples and add the classes there
